### PR TITLE
More verbose options

### PR DESCRIPTION
**Pull request by max2697**
@tronikos What do you think about this? It's a little bit ugly, but I don't have any better ideas for now. Maybe adding more constants for the `-v` and `-vv` options?

### CODE REVIEW

**Comment by max2697** [comment]
P.S. There are some issues with the opower file and git. I am checking it

File: README.md [hunk 1]
```diff
@@ -80,8 +80,8 @@ pytest
 
 # Run demo
 python src/demo.py --help
-# To output debug logs to a file, change DEBUG_LOG_RESPONSE to True in opower.py and run:
-python src/demo.py --verbose 2> out.txt
+# To output debug logs to a file run:
```
Comment (tronikos) [inline README.md:83]: `# To output debug logs and API responses to a file run:`

**Review by tronikos — COMMENTED** [review]
Looks good. Thanks! I manually diffed opower.py. Once you fix the diff I can merge. Feel free to start over in a new PR.

File: README.md (hunk 1 above)
Comment (max2697) [reply README.md]: Updated. I fixed diff and also updated logging for smud provider

**Review by tronikos — APPROVED** [review]
Thanks!

**Pull request merged**